src/CMakeLists.txt

```diff
@@ -214,6 +214,10 @@ ELSE()
   ENDIF()
 ENDIF()
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
CMake 3.1 came out in 2014, I don't think it is a problem to upgrade to it. I'm sure we can find a PPA for Travis to use.

I don't want to add workarounds for ancient software, let's start using the new stuff!